conda/environments/all_cuda-118_arch-x86_64.yaml

```diff
@@ -13,6 +13,7 @@ dependencies:
 - ninja
 - numba>=0.57
 - numpy>=1.21
+- nvcc_linux-64=11.8
```

Comment:
I don't think this change is strictly necessary, but I suppose it's worth partially aligning the 11 and 12 by including the compiler compat package for 11.

Reply:
All RAPIDS packages should be including nvcc in their dependencies.yaml. This has been done for most (cuDF, cuML, ...) but not yet for RMM. I took this opportunity to align it since we're adjusting compilers.